build.gradle

```diff
@@ -61,9 +61,9 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3'
 
     implementation "org.slf4j:slf4j-api"
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.2'
-    testImplementation 'org.mockito:mockito-junit-jupiter:5.10.0'
-    testImplementation 'org.mockito:mockito-core:5.10.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter:5.10.3'
```

Comment:
note: looks like junit 5.11.0 is available

Reply:
I don't see 5.11.0 on the https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter

Reply:
I have just updated the version in th2 gradle plugin. Could you please try to update the version and if it is found by Gradle - commit the update?

Reply:
the 5.11.0 version found, thank you.